apcd-cms/src/apps/registrations/views.py

```diff
@@ -25,7 +25,7 @@ class RegistrationFormView(TemplateView):
     def get(self, request):
         formatted_reg_data = []
         renew = False
-        reg_id = request.GET.get('reg_id', None)
+        reg_id = request.GET.get('reg_id', None).strip('/')  # reg_id coming from renew has trailing slash appended, need to remove to pass correct request through
```

Comment:
To remove trailing slash, should this be rstrip ? 
```suggestion
        reg_id = request.GET.get('reg_id', None).rstrip('/')  # reg_id coming from renew has trailing slash appended, need to remove to pass correct request through
```